modules/web/js/ballerina/visitors/source-gen/package-definition-visitor.js

```diff
@@ -47,10 +48,11 @@ class PackageDefinitionVisitor extends AbstractSourceGenVisitor {
 
     endVisitPackageDefinition(packageDefinition) {
         if (!_.isNil(packageDefinition.getPackageName()) && packageDefinition.getPackageName() !== '') {
+            let tailingWS = packageDefinition.getParent().getWSRegion(3);
+            tailingWS = (packageDefinition.whiteSpace.useDefault) ? '\n\n' : tailingWS;
```

Comment:
We are completely reassigning the value for tailingWS,   should this be 'tailingWS += '  ?